s3.tf

```diff
@@ -23,6 +23,7 @@ module "s3" {
 
 
 resource "aws_s3_bucket_policy" "docs" {
+  count  = length(module.s3.s3_bucket_id) > 0 ? 1 : 0
```

Comment:
@franklinpashok Plan failed when i tried the example. It gives the error "The "count" value depends on resource attributes that cannot be determined until apply"

Just thinking whether we can depend on the **create_bucket** and depends_on with module.s3 instead of count.

Reply:
updated